### PR TITLE
feat(share): add telegram

### PR DESCRIPTION
**Pull request by pidupuis**
Fix and complete Telegram sharing link

### CODE REVIEW

**Comment by AkarinServer** [comment]
@pidupuis 我靠这个改动很大啊。。。先看看，然后注意合并到canary分区先

File: layout/_partial/isolate-sns_list.ejs [hunk 1]
```diff
@@ -62,4 +62,12 @@
 		<i class="fa fa-linkedin fa-lg" aria-hidden="true"></i>
 	</a>
     <% } %>
+
+    <!-- Telegram -->
+    <% if(theme.sns.linkedin == undefined){ %>
+    <% }else{ %>
```
Comment (cubesky) [inline layout/_partial/isolate-sns_list.ejs]: I think this should change to `theme.sns.telegram` .

**Comment by pidupuis** [comment]
Yes sure, sorry for the bad copy/paste. I'll close this pull request and create another one ;)

**Comment by viosey** [comment]
@pidupuis You needn't create another one, commit the correct version to this PR again is fine.

**Comment by pidupuis** [comment]
@viosey No problem, I prefer to follow the rules ^^
https://github.com/viosey/hexo-theme-material/pull/112

**Pull request closed without merging**